build/dockerfiles/rhel.Dockerfile

```diff
@@ -81,7 +81,7 @@ RUN sed -i /etc/httpd/conf/httpd.conf \
     -e "s,logs/error_log,/dev/stderr," \
     -e "s,logs/access_log,/dev/stdout," \
```

Comment:
My understanding is that these lines configure httpd to log to stderr/stdout instead of a logfile.

Reply:
well, then maybe on rhel they're not working.